zh/enable-tls-for-mysql-client.md

```diff
@@ -419,6 +532,63 @@ spec:
    name: ${cluster_name}
  initSql: |-
    create database app;
+ tlsClientSecretName: ${cluster_name}-tidb-initializer-client-secret
+```
+
+backup.yaml:
+
+```
+apiVersion: pingcap.com/v1alpha1
+kind: Backup
+metadata:
+  name: ${cluster_name}-backup
+  namespace: ${namespace}
+spec:
+  backupType: full
+  br:
+    cluster: ${cluster_name}
+    clusterNamespace: ${namespace}
+    sendCredToTikv: true
+  from:
+    host: ${host}
+    secretName: ${tidb_secret}
+    port: 4000
+    user: root
+    tlsClientSecretName: ${cluster_name}-backup-client-secret
+  s3:
+    provider: aws
+    region: ${my_region}
+    secretName: ${s3_secret}
+    bucket: ${my_bucket}
+    prefix: ${my_folder}
+```
+
+restore.yaml:
+
+```
+apiVersion: pingcap.com/v1alpha1
+kind: Restore
+metadata:
+  name: ${cluster_name}-restore
+  namespace: ${namespace}
+spec:
+  backupType: full
+  br:
+    cluster: ${cluster_name}
+    clusterNamespace: ${namespace}
+    sendCredToTikv: true
+  to:
+    host: ${host}
+    secretName: ${tidb_secret}
+    port: 4000
+    user: root
+    tlsClientSecretName: ${cluster_name}-restore-client-secret
+  s3:
+    provider: aws
+    region: ${my_region}
+    secretName: ${s3_secret}
+    bucket: ${my_bucket}
+    prefix: ${my_folder}
 ```
 
 其中 `${cluster_name}` 为集群的名字，`${namespace}` 为 TiDB 集群部署的命名空间。通过设置 `spec.tidb.tlsClient.enabled` 属性为 `true` 来开启 MySQL 客户端 TLS。
```

Comment:
PTAL at L596 ```将上面文件保存为 `cr.yaml` ```. There are 3 yaml files. Please specify which file shall be saved.